src/resources/help/en_US/relnotes3.3.4.html

```diff
@@ -26,6 +26,7 @@
   <li>Fixed a bug where an error would incorrectly be detected when creating ghost zones. Specifically, it occurred when a block had mixed materials and the block didn't contain the material the variable was defined on.</li>
   <li>Fixed a bug reading particle data from Mili files. The particle data would not appear and could cause the engine to crash if the number of particles was larger than the number of elements.</li>
   <li>Fixed a bug where the pseudocolor plot would have NaNs in the legend if the mesh contained no elements or all the elements were ghost elements. Now it preints an error message with a description of the error.</li>
+  <li>Fixed a bug with 'LaunchNowin' on Windows.</li>
```

Comment:
```suggestion
  <li>Fixed a bug where 'LaunchNowin' would fail on Windows when additional launch arguments were specified with `AddArgument`.</li>
```